### PR TITLE
#73659 Add Some Clarifying Text to Per-Page Option Label

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**#73659**](http://central.tri.be/issues/73659)

### CODE REVIEW

File: src/admin-views/tribe-options-general.php [hunk 1]
```diff
@@ -25,6 +25,12 @@
 	)
 );
 
+$postsPerPage_tooltip = esc_html__( 'The number of events per page on the List View. Does not affect other views.', 'the-events-calendar' );
```
Comment (bordoni) [inline src/admin-views/tribe-options-general.php]: Avoid `camelCased` variables

**Comment by ggwicz** [comment]
Not sure why I did that, I never camel-case...but anyways, I've fixed that and this is ready for another review! 🔍

**Comment by ggwicz** [comment]
Fixed merge conflicts; this one's ready for another review 🔍

**Comment by ggwicz** [comment]
Addressed readme conflict, ready for another review 🔍

**Pull request merged**